packages/strapi-connector-bookshelf/lib/queries.js

```diff
@@ -94,7 +94,7 @@ module.exports = function createQueryBuilder({ model, strapi }) {
     const runCreate = async trx => {
       // Create entry with no-relational data.
       const entry = await model.forge(data).save(null, { transacting: trx });
-      const isDraft = hasDraftAndPublish && !entry.published_at;
+      const isDraft = contentTypesUtils.isDraft(entry.toJSON(), model);
```

Comment:
Why do you suddenly need to do call `.toJSON()`?

Reply:
It's a fix, it was working properly before